LBaaS/index.php

```diff
@@ -0,0 +1,49 @@
+<html>
+<head>
+<meta content="text/html; charset=ISO-8859-1" http-equiv="content-type"><title>Home</title>
+<link href='https://fonts.googleapis.com/css?family=Poiret One' rel='stylesheet' type='text/css'>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+</head>
+<body>
+
+<div class=container>
+<div class=page-header>
+<div style="font-family: Poiret One; font-size: 1.5em; letter-spacing: 0.3em;">
+
+<?php
+echo "<br>";
+echo "<h1> AnyApp </h1>";
+echo "<br>";
+session_start();
+echo "<br>";
+if (isset($_SESSION['upstream-servers'])) {
```

Comment:
we can leave out the whole session section, since we don't use a shared session in this example

Reply:
I wanted to leave the anyapp "as is". So we don't need to update it's code if we use it again.
It could also make sense to post it on github and download it to any instance we deploy. So it doesn't have to be uploaded over and over again.